Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,24 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    expand = File.expand_path("~/Applications")
+    File.exist?("#{expand}/Firefox.app")
```

Comment:
I think its a better style to merge these two lines: 
`File.exist? File.expand_path("~/Applications/Firefox.app")`